src/components/CR8 - GalleryText/GalleryText.vue

```diff
@@ -0,0 +1,234 @@
+                :key="`${index}-${company.name}`" 
+                class="company-link"  
+                :to="`/${company.name}`" tag="span"
+            >
+                <component 
+                    :is="`logo-${company.name}`"
+                    class="company-logo"
+                    @mouseover="set(company)" 
+                    @mouseout="set(null)" />
+                {{company.text}}
+            </nuxt-link>
+        </div>
+
+        <!-- Images absolute -->
+        <transition
+            v-for="(image, index) in images"
+            :key="image.id"
+            name="fade"
+            mode="out-in"
+        >
+            <wp-image
+                v-show="index == activeImage"
+                class="image"
+                :image="image"
+                mode="fullbleed"
+            />
+        </transition>
+
+        <!-- Bottom Text -->
+        <p class="end-text">These diverse talent are connected by a creative singularity.</p>
+        <p class="end-text">Together we are <logo-makemake class="company-logo" /></p>
+
+    </div>
+</template>
+
+<script>
+import WpImage from "@/components/global/WpImage"
+import NuxtLink from "@/components/global/NuxtLink"
+import SvgArrowRight from "@/assets/svgs/arrow-right.svg"
+
+// Company Logos
+import LogoRockPaperScissors from '@/assets/svgs/companies/logo-rock-paper-scissors.svg'
+import LogoElastic from '@/assets/svgs/companies/logo-elastic.svg'
+import LogoA52 from '@/assets/svgs/companies/logo-a52.svg'
+import LogoPrimary from '@/assets/svgs/companies/logo-primary.svg'
+import LogoJax from '@/assets/svgs/companies/logo-jax.svg'
+import LogoIndestructible from '@/assets/svgs/companies/logo-indestructible.svg'
+
+import LogoMakemake from '@/assets/svgs/companies/logo-makemake.svg'
+
+export default {
+    components: { 
+        WpImage, NuxtLink, SvgArrowRight, 
+        //Company Logos
+        LogoRockPaperScissors, LogoElastic, LogoA52, LogoPrimary, LogoJax, LogoIndestructible, LogoMakemake
+    },
+    props: {
+        companies: {
+            type: Array,
+            default: () => []
+        }
+    },
+    data () {
+        return {
+            company: null,
+            timer: null,
+            activeImage: 0
+        }
+    },
+    computed: {
+        images() {
+            return this.company ? this.company.images : []
+        },
+        imagesLength () {
+            return this.images.length
+        }
+    },
+    methods: {
+        set(company) {
+            if (!company) return this.reset()
+            this.company = company
+            this.start()
+        },
+        start() {
+            if (!this.timer) {
+                this.timer = setInterval( () => {                    
+                    return this.activeImage < this.imagesLength
+                        ? this.activeImage = this.activeImage + 1 
+                        : this.activeImage = 0
+                }, 500)
+            }
```

Comment:
A better way to do this:

```
     clearInterval(this.timer)

     this.timer = setInterval( () => {                    
          if(this.activeImage < this.imagesLength) {
               this.activeImage = this.activeImage++ 
          } else {
              this.activeImage = 0
          }
     }, 800)
```

You don't need the return (it does that anyway) and I personally don't like ternaries, harder to understand IMHO. Just a matter of taste.